airflow/bin/cli.py

```diff
@@ -1364,6 +1373,12 @@ class CLIFactory(object):
                   "again."),
             type=float,
             default=1.0),
+        'reset_dag_run': Arg(
+            ("--reset_dagruns",),
+            ("if set this flag, we will delete all the success "
```

Comment:
wait I don't think that description is right, backfill does not "delete the success tasks". Maybe something more like "if set, the backfill will delete existing backfill-related DAG runs and start anew with fresh, running DAG runs"

Reply:
hey Max, just want to make sure I understand your point correctly. since task_instances are shared by backfill runs and daily scheduled run, I think we need to delete clear the task instances as well(https://github.com/apache/incubator-airflow/blob/master/airflow/models.py#L3738) in order for backfill to rerun for the given date range.